site/visitor-console/src/pages/PageNotFound.tsx

```diff
@@ -1,24 +1,24 @@
-import { Link } from "react-router-dom";
+import React from 'react';
+import { Link } from 'react-router-dom';
 
-const PageNotFound = () => {
-  return (
-    <div>
-      <p
-        style={{ textAlign: "center", color: "white", paddingBottom: "100px" }}
-      >
-        <h1 style={{ fontSize: "350%" }}>Error 404</h1>
-        <h2>Page Not Found</h2>
-      </p>
-      <p style={{ textAlign: "center" }}>
-        <Link
-          to="/"
-          style={{ color: "white", paddingBottom: "100px", fontSize: "200%" }}
-        >
-          Go Back to Home Page
-        </Link>
-      </p>
-    </div>
-  );
-};
+class NotFoundPage extends React.Component{
+    render(){
+        return <div className="container bg-primary p-5 rounded" style={{height: "400px"}}>
+            <p style={{textAlign:"center", color:"white", paddingBottom: "100px"}}>
+                <div className="text-center mb-4">
 
-export default PageNotFound;
+                    <h1 className="text-secondary fw-bold mb-1 text-center" style={{fontSize:"350%"}}>ERROR 404 :(</h1>
+                    <h2 className="text-white fw-bold mb-1 text-center">Page Not Found</h2>
+                </div>
+    
+            </p>
+            <div className="text-secondary fw-bold mb-4 text-center">
+                <p style={{textAlign:"center"}}>
+                    <Link to="/" style={{color:"white", paddingBottom: "50px", fontSize:"150%"}}>Go Back to Home Page</Link>
+                </p>
+            </div>
+
+          </div>;
+    }
+}
+export default NotFoundPage;
```

Comment:
This is nitpicky, but can you `git mv site/visitor-console/src/pages/PageNotFound.tsx site/visitor-console/src/pages/NotFoundPage.tsx` so that the filepath matches the component name? Code is write-once, read-many, and little things like this will hopefully make it easier for future devs to connect the dots quicker.

Reply:
I have made those changes in my latest commit.